contracts/wawaka/common/Cryptography.cpp

```diff
@@ -405,6 +405,43 @@ bool ww::crypto::ecdsa::generate_keys(
     return true;
 }
 
+/* ----------------------------------------------------------------- *
+ * NAME: ww::crypto::
+ * ----------------------------------------------------------------- */
+bool ww::crypto::ecdsa::generate_keys(
```

Comment:
generate to me would imply we are generating new key but we really (re)create (i.e., deserialize) existing keys?

Reply:
strictly... yes. in practice, we're creating keys in a form that can be used in other operations.